app/src/main/res/layout/activity_main.xml

```diff
@@ -1,48 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity">
-
+    layout_width="match_parent"
+    layout_height="match_parent">
 <LinearLayout
```

Comment:
Nice work adding the `ScrollView`! One nit: you could indent the `LinearLayout` over 4 spaces to show that it is within the `ScrollView`.